test/Elastic.Apm.Tests.MockApmServer/AssertValidExtensions.cs

```diff
@@ -217,7 +217,16 @@ internal static void AssertValid(this CapturedException thisObj)
 
 			thisObj.Code?.AssertValid();
 			thisObj.StackTrace?.AssertValid();
-			thisObj.Type.AssertValid();
```

Comment:
`Message` or `Type` is required. If `Message` is present `Type` can be null. 

https://github.com/elastic/apm-server/blob/master/docs/spec/errors/error.json#L98